src/model/event/osw-upload.ts

```diff
@@ -6,8 +6,6 @@ import { PolygonDto } from "../polygon-model";
 export class OswUpload extends AbstractDomainEntity {
     @Prop('tdei_org_id')
     tdeiOrgId?: string;
-    @Prop('tdei_record_id')
-    tdeiRecordId?: string;
```

Comment:
why is this property removed?

Reply:
That is part of Queue message. it was missed last time while refactoring